File: app/src/main/java/ru/dyatel/tsuschedule/layout/WeekFragmentPagerAdapter.java
package ru.dyatel.tsuschedule.layout;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import ru.dyatel.tsuschedule.ParityReference;
import ru.dyatel.tsuschedule.fragments.WeekFragment;

import java.util.Locale;

public class WeekFragmentPagerAdapter extends FragmentPagerAdapter {

	public WeekFragmentPagerAdapter(FragmentManager fm) {
		super(fm);
	}

	@Override
	public Fragment getItem(int position) {
		return WeekFragment.newInstance(ParityReference.getParityFromIndex(position));
	}

	@Override
	public int getCount() {
		return 2;
	}

	@Override
	public CharSequence getPageTitle(int position) {
		return ParityReference.getStringFromIndex(position).toUpperCase(Locale.getDefault());
	}

}

File: app/src/main/kotlin/ru/dyatel/tsuschedule/layout/WeekFragmentPagerAdapter.kt
package ru.dyatel.tsuschedule.layout

import android.app.Fragment
import android.app.FragmentManager
import android.content.Context
import android.support.v13.app.FragmentPagerAdapter
import ru.dyatel.tsuschedule.fragments.WeekFragment
import ru.dyatel.tsuschedule.data.indexToParity

class WeekFragmentPagerAdapter(fm: FragmentManager, private val context: Context) : FragmentPagerAdapter(fm) {

    override fun getCount() = 2

    override fun getItem(position: Int): Fragment = WeekFragment.getInstance(indexToParity(position))

    override fun getPageTitle(position: Int) = indexToParity(position).toText(context)

}